design/README.md

```diff
@@ -0,0 +1,52 @@
+# The Design Tools We Use and Why We Use Them
+
+## Sketch for creating comps 
+We use Sketch to create comps for websites. Sketch is particularly good at mocking up UIs because of how it blends vector and bitmap artwork. Comps created in Sketch are used to communicate a design vision to the client and our team, not necessarily to show exactly what a website will look like.
+
+We use a team-based license for Sketch. Talk with Jeremy to set that up. 
+
+## InVision for design communication 
+InVision acts as a hub for design-centric communication between us and clients. It provides a few different features that help us communicate design ideas: 
+
+- **Mood Boards:** Early in a design process, it can be helpful to collect a bunch of images that communicate a specific look and feel to aim for. InVision lets us upload images to a board and provides features for sharing them with our team and clients. 
+- **Syncing:** InVision lets us upload comps in order to get feedback from clients. This is the best way to make our Sketch files viewable by clients. The most convenient way to sync from Sketch to InVision is by using InVision's Craft plugin. https://www.invisionapp.com/craft 
```

Comment:
Is it possible to make the link from "InVision's Craft plugin" instead of including the raw URL?